### PR TITLE
Vue cdav copy addressbook

### PR DESCRIPTION
**Pull request by sleepypioneer**
Please check my code, not feeling so confident with this

### CODE REVIEW

File: src/components/Settings/SettingsAddressbook.vue [hunk 1]
```diff
@@ -222,24 +222,27 @@ export default {
...
 			this.copyLoading = true
+			event.stopPropagation()
+
 			// copy link for addressbook to clipboard
 			this.$copyText(this.addressbook.url).then(e => {
+				event.preventDefault()
 				this.copySuccess = true
 				this.copied = true
+				// timeout sets the text back to copy to show text was copied
+				setTimeout(() => {
```
Comment (skjnldsv) [inline src/components/Settings/SettingsAddressbook.vue]: Hum,timeout are bad feedback when they're not relevant. Here the string is copied right away but we get the feeling we need to wait :)

Comment (sleepypioneer) [reply src/components/Settings/SettingsAddressbook.vue]: Ok I just thought otherwise it really doesn't show as its pretty quick!

**Comment by skjnldsv** [comment]
@sleepypioneer I guess so yes :)
I did not noticed before, sorry :see_no_evil:

**Comment by sleepypioneer** [comment]
done, I also added notifications for copy and delete for if they were successful, otherwise I think it is unclear something happened.

File: src/components/Settings/SettingsAddressbook.vue [hunk 2]
```diff
@@ -171,19 +170,17 @@ export default {
...
 		},
 		downloadAddressbook() {
-			// change to loading status
-			this.downloadLoading = true
-			setTimeout(() => {
-				// stop loading status regardless of outcome
-				this.downloadLoading = false
-			}, 1500)
+			// Notify download started
+			OC.Notification.showTemporary(t('contacts', 'Downloading Addressbook'))
```
Comment (skjnldsv) [inline src/components/Settings/SettingsAddressbook.vue]: Not needed, if the download starts, the user already knows it ;)

Comment (sleepypioneer) [reply src/components/Settings/SettingsAddressbook.vue]: ok will remove it :) anything else before merging?

Comment (skjnldsv) [reply src/components/Settings/SettingsAddressbook.vue]: :see_no_evil: :grimacing: :rocket: GO

**Pull request merged**